### PR TITLE
Validation issues | 2 (BULK)

### PR DESCRIPTION
**Pull request by damabe**
This is a bulk update.

This PR is to fix issues related to build validation messages.

If these changes are not appropriate, please let me know. I'm only trying to clean up validation issues in this repo. So I can easily close this PR or modify it in whatever way is best for the content.

### CODE REVIEW

**Comment by damabe** [comment]
Global effort to fix build validation errors

@Lauragra @Danielabom

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

File: api-reference/v1.0/api/workbook-list-tables.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 ---
-title: "List tables"
-description: "Retrieve a list of table objects."
+title: "Retrieve a list of table objects"
+description: "In this article, you can learn how List tables retrieve a list of table objects."
```
Comment (Lauragra) [inline api-reference/v1.0/api/workbook-list-tables.md:3]: Global: Please revert the changes to the descriptions, if the intent here is to differentiate them to remove the duplicate description warnings. We have agreed that we will allow the duplicate descriptions across the versioned reference content.

File: api-reference/v1.0/api/workbook-list-tables.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 author: "lumine2008"
 ms.localizationpriority: medium
 ms.prod: "excel"
 doc_type: apiPageType
 ---
 
-# List tables
+# List table objects
```
Comment (Lauragra) [inline api-reference/v1.0/api/workbook-list-tables.md:10]: Global: We have specific requirements for topic titles, please don't update the H1 titles if the intent is to remove duplicate H1 title warnings. We append the version to the titles in search results via a global setting in the folder-level docfx files, so the version is reflected in the search results.

**Comment by damabe** [comment]
Closing the PR because it is all duplicate descriptions, titles, and H1s which can remain as is per reviewer comment.
